scss/_related-box.scss

```diff
@@ -34,11 +34,11 @@
 
 .n-content-related-box__headline,
 .n-content-info-box__headline {
-	@include oTypographySansBold($scale: 4, $progressive: true);
+	@include oTypographySans($weight: 'bold', $scale: 4);
 	font-size: 26px;
```

Comment:
Setting the size/line-height twice here

Reply:
could we run it by design to see if they'd agree to an on-piste sizing?

Reply:
I've opened an issue to track this, as it's not a regression I'm going to push on without it.

Reply:
that seems like the right choice